src/components/common/Modal/style.module.scss

```diff
@@ -37,12 +37,19 @@
     background: none;
   }
 
+  @media screen and (width <= vars.$breakpoint-md) {
+    &.bottomSheet {
+      padding: 4rem 0 0;
```

Comment:
![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/e131fb66-a6c1-4fa8-bbc3-15a46d54e1f7)

should also remove the bottom border radii when it's in bottom sheet mode